test/plugin/timezone.test.js

```diff
@@ -0,0 +1,131 @@
+import MockDate from 'mockdate'
+import moment from 'moment-timezone'
+import dayjs from '../../src'
+import utc from '../../src/plugin/utc'
+import timezone from '../../src/plugin/timezone'
+
+dayjs.extend(utc)
+dayjs.extend(timezone)
+
+beforeEach(() => {
+  MockDate.set(new Date())
+})
+
+afterEach(() => {
+  MockDate.reset()
+})
+
+const NY = 'America/New_York'
+
+describe('Guess', () => {
+  it('return string', () => {
+    expect(typeof dayjs.tz.guess()).toBe('string')
+  })
+})
+
+
+describe('Parse', () => {
+  it('parse target time string', () => {
+    const newYork = dayjs.tz('2014-06-01 12:00', NY)
+    const MnewYork = moment.tz('2014-06-01 12:00', NY)
+    expect(newYork.format()).toBe('2014-06-01T12:00:00-04:00')
+    expect(newYork.format()).toBe(MnewYork.format())
+    expect(newYork.utcOffset()).toBe(-240)
+    expect(newYork.utcOffset()).toBe(MnewYork.utcOffset())
+    expect(newYork.valueOf()).toBe(1401638400000)
+    expect(newYork.valueOf()).toBe(MnewYork.valueOf())
+  })
+
+  it('parse and convert between timezones', () => {
+    const newYork = dayjs.tz('2014-06-01 12:00', NY)
+    expect(newYork.tz('America/Los_Angeles').format()).toBe('2014-06-01T09:00:00-07:00')
+    expect(newYork.tz('Europe/London').format()).toBe('2014-06-01T17:00:00+01:00')
+  })
+})
+
+describe('Convert', () => {
+  it('convert to target time', () => {
+    const losAngeles = dayjs('2014-06-01T12:00:00Z').tz('America/Los_Angeles')
+    const MlosAngeles = dayjs('2014-06-01T12:00:00Z').tz('America/Los_Angeles')
```

Comment:
There seems to be a typo here, shouldn't this use `moment.tz` instead of `dayjs`?

Reply:
yep, ha